big.go

```diff
@@ -29,6 +29,26 @@ func NewBigIntFromInt64(v int64) *BigInt {
 	return &BigInt{new(big.Int).SetInt64(v)}
 }
 
+// NewBigIntFromString creates a BigInt by parsing a string.
+// nil will be returned if the given string is not parsable.
+func NewBigIntFromString(data string) *BigInt {
```

Comment:
If you look at `NewAddressFromHex` for example, you can always have an `error` as second return value, which translates to a Java Exception throw when it is not nil/null.

Reply:
Yeah also saw that but ignored it... but you're right @ggwpez it's cleaner to return `(*BigInt, error)` here.